pom.xml

```diff
@@ -1650,15 +1668,15 @@
         <orbit.version.commons.beanutils>1.8.0.wso2v1</orbit.version.commons.beanutils>
         <commons-io.wso2.version>2.4.0.wso2v1</commons-io.wso2.version>
         <commons-fileupload.version>1.1.1</commons-fileupload.version>
-        <commons-httpclient.version>3.0.1</commons-httpclient.version>
+        <orbit.version.commons.httpclient>3.1.0.wso2v6</orbit.version.commons.httpclient>
         <commons-dbcp.version>1.2.2</commons-dbcp.version>
         <commons-logging.version>1.1</commons-logging.version>
         <commons-lang.wso2.version>2.6.0.wso2v1</commons-lang.wso2.version>
         <jackson.version>1.9.2</jackson.version>
         <amber.version>0.22.1358727-wso2v2</amber.version>
         
         <!-- Carbon Commons -->        
-        <carbon.commons.version>4.5.4</carbon.commons.version>
+        <carbon.commons.version>4.6.51-SNAPSHOT</carbon.commons.version>
```

Comment:
Shall we upgrade to commons released version

Reply:
This branch (master_java11) is related to giving JDK 11 support for the products. So still the commons hasn't been released with JDK 11 support so we have used this snapshot version